### PR TITLE
Remove node customizations from fetch request

### PR DESCRIPTION
**Pull request by timothyfroehlich**
It is no longer used anywhere and was dead code.

Also pick up some top of file licenses that were missed.

### CODE REVIEW

File: codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt [hunk 1]
```diff
@@ -958,7 +958,7 @@ class BuilderProcessor(private val codeGenerator: CodeGenerator, val logger: KSP
             out.appendText("                customizations = customizations,\n")
             out.appendText("                modifier = modifier,\n")
             out.appendText(
-                "                serverParams = DocumentServerParams(queries, nodeCustomizations(), ignoredImages()),\n"
+                "                serverParams = DocumentServerParams(queries, ignoredImages()),\n"
```
Comment (rylin8) [inline codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt:961]: It's a bit more work, but are you planning on removing the code that generates the nodeCustomizations() function?

File: common/src/main/java/com/android/designcompose/common/GenericDocContent.kt [hunk 2]
```diff
@@ -52,6 +52,7 @@ class GenericDocContent(
             feedback.documentSaveError(error.toString(), docId)
         }
     }
+    @kotlin.ExperimentalUnsignedTypes
```
[the remark this answers is not in the document]
Comment (timothyfroehlich) [reply common/src/main/java/com/android/designcompose/common/GenericDocContent.kt:55]: Yeah, silences a warning

File: codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt (hunk 1 above)
Comment (timothyfroehlich) [reply codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt:847]: Good call, removing that now.

**Comment by timothyfroehlich** [comment]
> I think we want most of this function still. We don't need the part that generates the node customization code (lines 640-643), but I think the rest of it is to generate the JSON file that is used for our plugins and widget. 

@rylin How's that?

File: codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt [hunk 3]
```diff
@@ -548,104 +532,6 @@ class BuilderProcessor(private val codeGenerator: CodeGenerator, val logger: KSP
             }
         }
 
-        @OptIn(KspExperimental::class)
-        private fun visitFunctionNodeCustomizations(
-            function: KSFunctionDeclaration,
-            node: String,
-            isRoot: Boolean
-        ) {
```
[the remark this answers is not in the document]
Comment (rylin8) [reply codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt]: Ok looks good, can you just verify that the JSON output file is the same after your changes? If so I'm good with this.

[the remark this answers is not in the document]
Comment (timothyfroehlich) [reply codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt]: Checked Validation and Tutorial, they're the same.

**Review by rylin8 — APPROVED** [review]

**Pull request merged**